packages/solana/lib/src/rpc/json_rpc_client.dart

```diff
@@ -2,6 +2,7 @@ import 'dart:async';
 import 'dart:convert';
 
 import 'package:http/http.dart' as http;
+import 'package:http/http.dart';
```

Comment:
There's already imported `http`.

Reply:
It should help to have the linter work this for us but great catch :D